README.markdown

```diff
@@ -48,7 +48,9 @@ Note: On OSX, if you get `nvm: command not found` after running the install scri
 
 To verify that nvm has been installed, do
 
-    command -v nvm
+`source ~/.bash_profile` (or whichever profile you added nvm to)
```

Comment:
this needs to be `.`, not `source`, to be more portable - but also, if you have to manually source `nvm`, then it wasn't installed properly - so this step explicitly should *not* be included.